File: BackEnd-ProjectAPI/Sample/Sample.Infrastructure/EntityFramework/Company.cs
using Sample.Domain.Entities;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Sample.Infrastructure.EntityFramework
{
    public partial class Company
    {
        public int Id { get; set; }
        public string CompanyCode { get; set; } = null!;
        public string CompanyName { get; set; } = null!;
        public string? Address { get; set; }
        public string? ContactNo { get; set; }
        public string? Email { get; set; }
        public DateTime CreatedDateTime { get; set; } = DateTime.Now;
        public DateTime? ModifiedDateTime { get; set; }
        public bool IsActive { get; set; } = true;

        // Navigation property
        public virtual ICollection<Branch> Branches { get; set; } = new List<Branch>();
    }
}

File: FrontEnd-ProjectReact/my-react-app-inventory/src/types/company.ts
import type { Branch } from "./branch";

export interface Company {
  id: number;
  companyCode: string;
  companyName: string;
  address?: string;
  contactNo?: string;
  email?: string;
  createdDateTime: string; // or Date
  modifiedDateTime?: string; // or Date
  isActive: boolean;

  // Navigation property
  branches: Branch[]; // define this interface as well
}